front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -409,6 +408,7 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
               className="btn btn-primary"
               type="button"
               onClick={createStudy}
+              disabled={isStartDateValid || !isExpectedEndDateValid || !isActualEndDateValid}
```

Comment:
Missed the "!" here. Form is never valid for now :)
```suggestion
              disabled={!isStartDateValid || !isExpectedEndDateValid || !isActualEndDateValid}
```

Reply:
fixed !